### ISSUE TITLE
Question: How to add more than 1 publication

### ISSUE THREAD

**Issue by Rhinogradentia** [NBISweden/EMBLmyGFF3#19]
Hi,

I need to add more than one publication, how would this be possible? I already tried to reuse the --ra, --rt, --rl parameter for each publication, but only the last one will be used. Of course I could do it manually. but the RP field won't be filled automatically anymore and I would have to do it severeal times which can be very time consuming. 

Thank you very much in advance.

Best Regards,
Nadine

**Comment by Juke34** [comment]
Good catch, indeed you should be able to use those option for as many publication you would like to include. We will have a look at it and fix that as soon as we can.

**Comment by Rhinogradentia** [comment]
That would be really great - thank you!
